variables.tf

```diff
@@ -23,6 +23,11 @@ variable "users" {
   description = "A map with transfer users and configuration details"
 }
 
+variable "s3_id" {
```

Comment:
Can you add descriptions to the variables?

Reply:
Can you sort the variables? e.g. s3_id is before tags.

Reply:
done